sunpy/net/vso/attrs.py

```diff
@@ -230,102 +228,6 @@ def _create(wlk, root, api):
     })
 )
 
-# The idea of using a multi-method here - that means a method which dispatches
```

Comment:
You were able to simplify the attrs code?

Reply:
All this code was powering `vso.QueryResponse.search` and we removed that as it was broken since the 1.0 release.